curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8c4db0710f3260f867a92.md

```diff
@@ -30,13 +33,13 @@ assert.match(buyHealth.toString(), /if/);
 Your `if` statement should have the string `condition` for the condition.
```

Comment:
this hint text needs to change. We shouldn't check for the string condition. It should check for gold is greater than or equal to 10